arbiter/arbiter-ui/src/main/java/org/deeplearning4j/arbiter/ui/module/ArbiterModule.java

```diff
@@ -197,23 +195,27 @@ public synchronized void onAttach(StatsStorage statsStorage) {
             getDefaultSession();
     }
 
-    private Result currentSession() {
+    private void currentSession(RoutingContext rc) {
         String sid = currentSessionID == null ? "" : currentSessionID;
-        return ok(asJson(sid)).as(JSON);
+        rc.response()
+                .putHeader("content-type", "application/json")
+                .end(asJson(sid));
     }
 
-    private Result listSessions() {
-        return Results.ok(asJson(knownSessionIDs.keySet())).as(JSON);
+    private void listSessions(RoutingContext rc) {
+        rc.response()
+                .putHeader("content-type", "application/json")
+                .end(asJson(knownSessionIDs.keySet()));
     }
 
-    private Result setSession(String newSessionID) {
+    private void setSession(String newSessionID, RoutingContext rc) {
         log.debug("Arbiter UI: Set to session {}", newSessionID);
 
         if (knownSessionIDs.containsKey(newSessionID)) {
             currentSessionID = newSessionID;
-            return ok();
+            rc.response().end();
```

Comment:
Returning a status object might not be a bad idea here.

Reply:
You mean like `.setStatusCode(HttpResponseStatus.OK.code())`?
It's 200/ok by default... https://vertx.io/blog/some-rest-with-vert-x/